adding Milan chapter

pietro909 opened the pull request:
the brand new OTS chapter in Milano.

pietro909 committed:
adding Milan chapter

pietro909 committed:
my photo

pietro909 committed:
added pietro909

pietro909 committed:
full names updated

lehmannro commented on _cities/milan.md:
```diff
@@ -0,0 +1,17 @@
+  - alberto: Alberto Massa
+  - giorgia
+  - giorgio: Giorgio Mazzucchelli
+  - pietro909
```
Does this account actually exist?  Members without an opentechschool.org account should be added like the above, with `<image name>: <full name>` (otherwise it's going to add `pietro909@opentechschool.org` as a contact address for you.)

Also note that we'll happily give you accounts.  Could you drop emails to team.tech@opentechschool.org with your preferred user names and fallback emails?

pietro909 replied:
I'm registered as a forum user and in the mailing list since 2012. I don't know whether I should do something more.
For me is good to have the direct mail `pietro909@opentechschool` in order to keep the communication fluent.

lehmannro commented:
Could you add images for Alberto and Giorgio too?  If you don't have anything handy please copy `unkonwn.jpg`.

lehmannro commented on _cities/milan.md:
```diff
@@ -0,0 +1,17 @@
+title: Milan
+location: "Milan, Italy"
+tagline: "Pizza, code and fine wine!"
+mailing_list: team.milan
```
I couldn't find such a list in our domain.  Could you drop an email to team.tech@opentechschool.org so we don't forget about setting it up?

pietro909 committed:
missing photos for Alberto and Giorgio

pietro909 committed:
mailing list removed

pietro909 commented:
Thank you for pointing it out: I misunderstood the guidelines about names, sorry.
I've added missing photos and removed mailing list as we're not ready to manage it now.

gsambrotta committed:
add background image and about

gsambrotta committed:
add Kronos meetup page
